fake-snippets-api/lib/db/autoload-packages.json

```diff
@@ -1,13 +1,5 @@
 {
-  "snippets": [
-    {
-      "owner": "Abse2001",
-      "name": "Arduino-Nano-Servo-Breakout"
-    },
-    {
-      "owner": "ShiboSoftwareDev",
-      "name": "Wifi-Camera-Module"
-    },
```

Comment:
i like these packages!

Reply:
Will add those back, they were kind of failing to fetch majority of time. So removed it

Reply:
hmmm weird, maybe the package files are too big? I think these have `dist/circuit.json` so I wonder if that could be related